libusb/os/darwin_usb.h

```diff
@@ -29,74 +29,34 @@
 #include <IOKit/IOCFBundle.h>
 #include <IOKit/usb/IOUSBLib.h>
 #include <IOKit/IOCFPlugIn.h>
+#include <CoreServices/CoreServices.h>
```

Comment:
Why is CoreServices needed now?

Reply:
Probably was needed for some prior iteration of the code. Doesn't seem to be needed now.